aptly/interfaces.go

```diff
@@ -130,6 +130,8 @@ type Downloader interface {
 	DownloadWithChecksum(ctx context.Context, url string, destination string, expected *utils.ChecksumInfo, ignoreMismatch bool, maxTries int) error
 	// GetProgress returns Progress object
 	GetProgress() Progress
+	// GetLength of given url
```

Comment:
I think it should be better to say 'GetLength returns size by heading object with url` ?

It's not `len(url)` ;)